kasa/smartprotocol.py

```diff
@@ -86,20 +111,41 @@ async def _query(self, request: Union[str, Dict], retry_count: int = 3) -> Dict:
                     f"Unable to connect to the device: {self.host}: {cex}"
                 ) from cex
             except TimeoutError as tex:
-                await self.close()
-                raise SmartDeviceException(
-                    f"Unable to connect to the device, timed out: {self.host}: {tex}"
-                ) from tex
+                if retry >= retry_count:
+                    await self.close()
+                    raise SmartDeviceException(
+                        "Unable to connect to the device, "
+                        + f"timed out: {self.host}: {tex}"
+                    ) from tex
+                await asyncio.sleep(2)
```

Comment:
This sounds rather arbitrary, why 2 instead of 1 second? :-) It might be worth to constify this on top of the file.

Reply:
Made this a constant and set it to 1 for now. I seem to recall 2 was a good number when I was getting my HS100/XoR device to work after a timeout but I can't say for certain.